config/config.js

```diff
@@ -0,0 +1,2 @@
+exports.connectionString = process.env.DATABASE_URL;
+exports.connectionStringMainDB = process.env.DATABASE_URL_MAINDB;
```

Comment:
We don't need the second connection string for this project. For field-data service we needed it for backward comptability. For this project we just need it to write data to the treetracker schema alone.